cfme/common/provider.py

```diff
@@ -756,7 +756,7 @@ def _fill_credential(form, cred, validate=None):
                 'token_verify_secret': cred.verify_token,
                 'validate_btn': validate
             })
-            if validate:
+            if validate_hawkular:
```

Comment:
I suppose there could be several providers which use "token" credential. Please make sure that you change won't break other providers and tests.